src/ILCompiler.Compiler/src/IL/ILImporter.Scanner.cs

```diff
@@ -993,6 +1014,21 @@ private int ReadILTokenAt(int ilOffset)
                 + (_ilBytes[ilOffset + 3] << 24));
         }
 
+        private void ReportOutOfRangeBranchTarget(int targetOffset)
+        {
+            throw new TypeSystemException.InvalidProgramException();
+        }
+
+        private void ReportFallthrough()
```

Comment:
> Maybe the error condition can use longer or different name

Any suggestion? I don't particularly like that either, but I couldn't come up with anything better.